main.nf

```diff
@@ -2605,6 +2606,64 @@ process StrelkaBP {
 
 vcfStrelkaBP = vcfStrelkaBP.dump(tag:'Strelka BP')
 
+// STEP MSISENSOR.1 - SCAN
+
+// Scan reference genome for microsattelites
+process msisensorScan {
+    label 'cpus_1'
+    label 'memory_max'
+    // memory '20 GB'
+
+    tag {fasta}
+
+    input:
+    file(fasta) from ch_fasta
+    file(fastaFai) from ch_fai
+
+    output:
+    file "microsatellites.list" into msi_scan_ch
+
+    when: 'msisensor' in tools
+
+    script:
+    """
+    msisensor scan -d ${fasta} -o microsatellites.list
+    """
+}
+
+// STEP MSISENSOR.2 - SCORE
+
+// Score the normal vs somatic pair of bams
+
+process msisensor {
+    label 'cpus_4'
+    label 'memory_max'
+    // memory '10 GB'
+
+    tag {idSampleTumor + "_vs_" + idSampleNormal}
+
+    publishDir "${params.outdir}/MSI/${idSampleTumor}_vs_${idSampleNormal}/msisensor", mode: params.publishDirMode
+
+    input:
+    set idPatient, idSampleNormal, file(bamNormal), file(baiNormal), idSampleTumor, file(bamTumor), file(baiTumor) from pairBamMsisensor
+    file msiSites from msi_scan_ch
+
+    output:
+    set val("Msisensor"), idPatient, file("${idSampleTumor}_vs_${idSampleNormal}_msisensor"), file("${idSampleTumor}_vs_${idSampleNormal}_msisensor_dis"), file("${idSampleTumor}_vs_${idSampleNormal}_msisensor_germline"), file("${idSampleTumor}_vs_${idSampleNormal}_msisensor_somatic") into msisensor_out_ch
```

Comment:
Can you please add some information about the outputs in the docs?
https://github.com/nf-core/sarek/blob/dev/docs/output.md

Reply:
done in c1af38b97335644aaec2f2184b6463ac2d87f149